torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -121,16 +125,16 @@ def __init__(
         # Whether to reduce-scatter or all-reduce gradients, respectively
```

Comment:
Sorry, I realized that my previous suggestion changed the semantic that I originally had in mind back when I wrote this. With the new way, I think we want something like this:
```suggestion
        # Whether to reduce gradients at all (whether for FSDP or HSDP)
        self.reduce_grads: bool = True
        # Whether to all-reduce gradients for HSDP; only used if
        # `self.reduce_grads` is true, in which case setting this to false
        # means reduce-scatter but no all-reduce
```